litmus-portal/graphql-server/pkg/file_handlers/file_handlers.go

```diff
@@ -43,6 +46,17 @@ func FileHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if os.Getenv("PORTAL_SCOPE") == "cluster" {
+		portalEndpoint, err = k8s.GetPortalEndpoint()
+		if err != nil {
+			log.Print(err)
+		}
+	} else if os.Getenv("PORTAL_SCOPE") == "namespace" {
+		portalEndpoint = os.Getenv("PORTAL_ENDPOINT")
+	}
```

Comment:
shouldn't these be user entry? @rajdas98

Reply:
It's an user entry, I already put these envs in the manifest